baseapp/baseapp.go

```diff
@@ -27,6 +28,7 @@ type BaseApp struct {
 	// initialized on creation
 	Logger     log.Logger
 	name       string               // application name from abci.Info
+	codec      *wire.Codec          // Amino codec
```

Comment:
let's stick to `cdc` ? that's everywhere really

Reply:
Updated to `cdc`.